test/functional/Definition.hs

```diff
@@ -2,63 +2,19 @@ module Definition (tests) where
 
 import           Control.Lens
 import           Language.LSP.Types.Lens
-import           System.Directory
 import           Test.Hls
 import           Test.Hls.Command
 
 tests :: TestTree
-tests = testGroup "definitions" [
-
-    ignoreTestBecause "Broken: file:///Users/jwindsor/src/haskell-language-server/test/testdata/References.hs" $
-       testCase "goto's symbols" $ runSession hlsCommand fullCaps "test/testdata" $ do
-        doc <- openDoc "References.hs" "haskell"
-        defs <- getDefinitions doc (Position 7 8)
```

Comment:
you probably need to insert a `waitForAllProgressDone` call after opening the first document